include/blast/math/dense/DynamicMatrixPointer.hpp

```diff
@@ -220,7 +229,7 @@ namespace blast
     }
 
 
-    template <bool AF, typename MT>
+    template <bool AF, Matrix MT>
```

Comment:
Just a question: What does AF stand for?

Reply:
Alignment Flag